### DISCUSSION TITLE
unable to verify my edu identity

### DISCUSSION THREAD

**Discussion by pseudorandom314** [community/community#138437] [GitHub Education]
### Hi everyone,

I am studying for my remote CS degree from overseas at my school, and I believe I uploaded all the required documents, including my name, degree, time, and class registration as online, also added my name to billing page, but it still got rejected

**Comment by matheusaudibert** [comment]
Hello pseudorandom314,  one possible reason for the rejection could be that your school isn’t currently part of GitHub’s Student Developer Pack program. I’d recommend reaching out to GitHub support for more information or checking if your school is listed as eligible. Hope this helps!

**Reply by pseudorandom314** [reply]
Hey Matheus,

**Individual manual reviews and status updates are no longer provided.**
when I try to open a ticket from https://support.github.com/contact/education

and I found some of my classmate have edu pack enabled, so my school should be eligible for it.
not sure what should I try next

**Comment by davevad93** [comment]
Hi @pseudorandom314 ,

See the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352), there you'll find a lot of information that might be helpful to solve your application issues.
